implementations/rust/ockam/ockam_api/src/util.rs

```diff
@@ -412,12 +413,12 @@ pub mod test {
             NodeManagerGeneralOptions::new(cli_state.clone(), node_name, true, None),
             NodeManagerProjectsOptions::new(None, None, Default::default(), None),
             NodeManagerTransportOptions::new(
-                (
-                    crate::nodes::models::transport::TransportType::Tcp,
-                    crate::nodes::models::transport::TransportMode::Listen,
-                    "127.0.0.1".into(),
-                    "".into(),
-                ),
+                ApiTransport {
+                    tt: crate::nodes::models::transport::TransportType::Tcp,
+                    tm: crate::nodes::models::transport::TransportMode::Listen,
+                    socket_address: "127.0.0.1:123".into(),
```

Comment:
You forgot to parse the string here: `socket_address: "127.0.0.1:123".parse.unwrap(),`

Reply:
Intersting, how did it even managed to compile 🤔 😅